index.js

```diff
@@ -67,8 +67,48 @@ function toggleColor(element) {
     element.style.color = "red"
   }
 }
+let header = document.querySelector("h1#header")
+
+header.addEventListener("click", function() {
+  toggleColor(header)
+})
 
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('.like-button') // access 'like-button'
+let likesShow = document.querySelector('.likes') //access <p class="likes"> parseInt, then likes++
+
+// function addLike() {
+//   const likesNum = parseInt(likesShow.textContent)
+//   likesShow.textContent = likesNum ++ + "Likes"
+// }
+
+likeButton.addEventListener("click", function() {
+  let likesNum = parseInt(likesShow.textContent)
+  likesShow.textContent = likesNum + 1 + " Likes"
+})
```

Comment:
The key here is that you have to read some information from the DOM to get the current number of likes, increment it, and then write it back to the DOM. Nice work!